### PR TITLE
Create coinmarketcap.markdown

### PR DESCRIPTION
**Pull request by metbril**

### CODE REVIEW

File: alerts/coinmarketcap.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Coinmarketcap public API deprecated"
```
Comment (klaasnicolaas) [inline alerts/coinmarketcap.markdown:2]:
```suggestion
title: Coinmarketcap public API deprecated
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: alerts/coinmarketcap.markdown (hunk 1 above)
Comment (metbril) [reply alerts/coinmarketcap.markdown]: Fixed

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks @metbril 👍

**Comment by metbril** [comment]
Xref
https://github.com/home-assistant/core/issues/32551

**Pull request merged**